metaseq/modules/transformer_layer.py

```diff
@@ -37,65 +37,26 @@ def _ffn(x, fc1, activation_fn, fc2, dropout_module, ffn_ln=None):
         assert fc1.skip_bias_add
         x, bias_fc1 = fc1(x)
         x = fused_bias_gelu(x, bias_fc1)
-        if ffn_ln is not None:
```

Comment:
We may also regret this. Sync offline

Reply:
This was brought in with normformer which we turned off entirely for OPT. Do you want to add this one LN back?